packages/manager/apps/pci-private-registry/public/translations/ip-restrictions/Messages_fr_FR.json

```diff
@@ -0,0 +1,20 @@
+{
+  "private_registry_noCIDR": "Actuellement votre Managed Private Registry est ouverte à toutes les connexions. \nPour limiter l'accès à votre Managed Private Registry, il est important de définir des adresses IP autorisées (Bloc CIDR). \nEn procédant ainsi, vous assurez que seules les personnes ou les systèmes que vous avez approuvés peuvent accéder à votre Registry, ce qui renforce la sécurité et facilite la gestion des connexions.",
+  "ip_restrictions_add_block": "Ajouter un bloc CIDR autorisé",
+  "ip_restrictions_delete_block": "Supprimer un bloc CIDR autorisé",
+  "ip_restrictions_delete_multiple_block": "Supprimer plusieurs blocs CIDR autorisés",
+  "private_registry_bloc_cidr": "Bloc CIDR",
+  "private_registry_cidr_description": "Description",
+  "private_registry_cidr_authorization": "Composant autorisé",
+  "private_registry_cidr_manage_title": "Gérer les blocs CIDR autorisés",
+  "private_registry_cidr_authorization_select": "Sélectionner un composant",
+  "private_registry_cidr_validation_ipBlock": "Ce bloc doit respecter le format CIDR (par exemple : 192.168.0.0/24).",
+  "private_registry_cidr_validation_authorization": "Veuillez sélectionner au moins un composant autorisé.",
+  "private_registry_cidr_submit_success": "Votre bloc CIDR a été ajouté aux blocs CIDR autorisés.",
+  "private_registry_cidr_delete_success": "Votre bloc CIDR a été supprimé avec succès.",
+  "private_registry_cidr_delete_all_success": "Tous les blocs CIDR sélectionnés ont été supprimés avec succès.",
+  "private_registry_cidr_help_authorized_component": "<p>Il existe 2 composants logiciel pour gérer les blocs<br/> CIDR :</p><p style='display: block;'>Management: permet d'autoriser l'acces à UI et à <br/> l'API Harbor.</p><p style='display: block;'>Registry: permet aux différents client / CLI d'autoriser <br/>  l'acces au backend Registry (Docker, Helm, ... )</p>",
```

Comment:
This sentence can be broken down into multiple sub-sentences and avoid HTML elements in translations.